templates/web/base/report/update-form.html

```diff
@@ -1,5 +1,5 @@
 [% allow_creation = !c.cobrand.only_authed_can_create || (c.user && c.user.from_body) %]
-[% RETURN IF NOT allow_creation %]
+[% RETURN IF NOT allow_creation OR problem.extra.closed_updates %]
```

Comment:
Does this check need to go into the Bromley `update-form.html` too?

Reply:
Oh yes. Will steal refactor commit from other open PR that refactors new and add update to that.